admin/view-translations-term.php

```diff
@@ -75,8 +82,8 @@
 					esc_attr( $language->slug ),
 					/* translators: accessibility text */
 					esc_html__( 'Translation', 'polylang' ),
-					( empty( $translation ) ? 0 : esc_attr( $translation->term_id ) ),
-					( empty( $translation ) ? '' : esc_attr( $translation->name ) ),
+					( ! $is_existing_translation ? '0' : esc_attr( (string) $translation->term_id ) ),
+					( ! $is_existing_translation ? '' : esc_attr( $translation->name ) ),
```

Comment:
This order is more convenient to read:
```suggestion
					$translation_exists ? (int) $translation->term_id : 0,
					$translation_exists ? esc_attr( $translation->name ) : '',
```
NB: The first line supposes that we use `%3$d` for the value.

Reply:
Done in e6e41a36cf9ff7fd0a30ec02e2b1c40b0e44eeb7 for `printf` pattern.